sources/src/resources/init-scripts/gradle-actions.inject-develocity.init.gradle

```diff
@@ -107,21 +108,35 @@ if (GradleVersion.current() < GradleVersion.version('6.0')) {
                     logger.quiet("Applying $BUILD_SCAN_PLUGIN_CLASS via init script")
                     applyPluginExternally(pluginManager, BUILD_SCAN_PLUGIN_CLASS)
                     if (geUrl) {
-                        logger.quiet("Connection to Develocity: $geUrl, allowUntrustedServer: $geAllowUntrustedServer")
+                        logger.quiet("Connection to Develocity: $geUrl, allowUntrustedServer: $geAllowUntrustedServer, captureTaskInputFiles: $geCaptureTaskInputFiles")
                         buildScan.server = geUrl
                         buildScan.allowUntrustedServer = geAllowUntrustedServer
                     }
                     buildScan.publishAlways()
+                    if (isAtLeast(gePluginVersion, '2.1') && GradleVersion.current() >= GradleVersion.version('5.0')) {
```

Comment:
There appears to be an existing pattern of defining an `atLeastGradleX` variable and referencing that throughout the script. See here where it's present in the `initscript` block:

https://github.com/gradle/actions/blob/571dd492c58a7353d37034c0b80121768b0a19b5/sources/src/resources/init-scripts/gradle-actions.inject-develocity.init.gradle#L27-L28 

And here where it's present at the root level:

https://github.com/gradle/actions/blob/571dd492c58a7353d37034c0b80121768b0a19b5/sources/src/resources/init-scripts/gradle-actions.inject-develocity.init.gradle#L89

So I suggest creating an `atLeastGradle5` next to the `atLeastGradle4` at the root level.

Reply:
Done